src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/constants/Status.kt

```diff
@@ -16,14 +16,22 @@ enum class Status(
   val nomisStatus: String?,
 ) {
   DRAFT("Draft", null),
-  AWAITING_ANALYSIS("Awaiting analysis", "AWAN"),
-  IN_ANALYSIS("In analysis", "INAN"),
-  INFORMATION_REQUIRED("Information required", "INREQ"),
-  INFORMATION_AMENDED("Information amended", "INAME"),
+  AWAITING_REVIEW("Awaiting review", "AWAN"),
+  ON_HOLD("On hold", "INAN"),
+  NEEDS_UPDATING("Needs updating", "INREQ"),
+  UPDATED("Updated", "INAME"),
   CLOSED("Closed", "CLOSE"),
+
+  // TODO: POST_INCIDENT_UPDATE/INCIDENT_UPDATED will be removed
   POST_INCIDENT_UPDATE("Post-incident update", "PIU"),
   INCIDENT_UPDATED("Incident updated", "IUP"),
+
   DUPLICATE("Duplicate", "DUP"),
+
+  // TODO: New statuses tbc
```

Comment:
i think we should take this moment to reorder statuses into some kind of order. do u think this is as sensible as can be?

Reply:
Tbh when I had a think at this some time ago I ended up with the conclusion there isn't such a natural order aside from "DRAFT" is first, "CLOSED" is last 😅 - anything else could really happen in almost any order so I am not quite sure.

I'm tempted to think, it doesn't matter (It shouldn't right?)

> do u think this is as sensible as can be?

I think it kinda does actually, the first block is pretty much the right order, then you got the 2 statuses that will go at some point and then you have the "not reportable" statuses which I think they can go after "CLOSED".